### ISSUE TITLE
PWA vs Native App

### ISSUE THREAD

**Issue by devagx** [TR-SUNSHINE/Static-Frontend#23]
# Identify what features we need to implement and to who e.g. android\ios\both
-ADV and DISADV of PWA
-DISADV - Restricted hardware access to IOS
-DISADV - No push notifications for IOS
-DISADV - bigger drain on battery power
# Identify based on above if PWA still fits based on what the app needs to deliver, but also taking into account it specifically states on brief “Progressive web app”.

**Comment by Chenerywoman** [comment]
Hi @devagx @RanaAlqrenawi @devagx This is a big topic to discuss - thanks for your notes above.  I have done the code along and read the articles and also have questions which I will try to find time to write down so we can discuss Progressive Apps at Sprint Planning.  

I think we have to do some kind of PWA functionality as it is clearly in the brief, although I feel we need to understand more what would work - the code along video just had on the offline version : :check the weather when back online???!

Can we move this discussion over to the React one as we have stopped work on the static repository.

**Comment by devagx** [comment]
yeah sorry, was meant to put it in the other project. will move this
